articles/azure-functions/functions-identity-based-connections-tutorial-2.md

```diff
@@ -97,8 +97,15 @@ You've granted your function app access to the service bus namespace using manag
 
 1. After you create the two settings, select **Save** > **Confirm**.
 
+> [!NOTE]
+> When using [Azure App Configuration](../../articles/azure-app-configuration/quickstart-azure-functions-csharp.md) to provide settings for Managed Identity connections, setting names should use a valid key separator such as `:` or `/` in place of the `__` to ensure names are resolved correctly.
```

Comment:
I believe this also applies to KeyVault configuration.

Reply:
Good call. Will update to reference KeyVault too.